### PR TITLE
Return the result of send_mail for check the number of sent emails

### PR DESCRIPTION
**Pull request by gaamdalurt**
So, this is my first pull request. If I have made mistakes, please help me.

Basically, send_email function return a value to test on, email_user is not. It will be useful to return the same value to known the number of sent emails.

### CODE REVIEW

File: django/contrib/auth/models.py [hunk 1]
```diff
@@ -377,8 +377,8 @@ def get_short_name(self):
         return self.first_name
 
     def email_user(self, subject, message, from_email=None, **kwargs):
-        """Send an email to this user."""
-        send_mail(subject, message, from_email, [self.email], **kwargs)
+        """Send an email to this user and return the number of sent emails."""
```
Comment (codingjoe) [inline django/contrib/auth/models.py]: My spellcheck tool of choice (languagetool) suggests a little change.
```suggestion
        """Email this user and return the number of emails sent."""
```

**Comment by felixxm** [comment]
@gaamdalurt Thanks for this patch :+1: Accepted [ticket](https://code.djangoproject.com/newticket), tests and docs changes are also required.

**Comment by gaamdalurt** [comment]
> @gaamdalurt LGTM, but I would suggest adding a little release not to `3.1.txt`.

Absolutely, how can I change that? :)

> @gaamdalurt Thanks for this patch  Accepted [ticket](https://code.djangoproject.com/newticket), tests and docs changes are also required.

Oh, sorry. So, I should open a ticket follow your link before, I see.

**Comment by felixxm** [comment]
You should add release notes to (`docs/releases/3.1.txt`) in the `django.contrib.auth` section and change docs in [django.contrib.auth.models.User.email_user](https://docs.djangoproject.com/en/dev/ref/contrib/auth/#django.contrib.auth.models.User.email_user), e.g.
```
    .. method:: email_user(subject, message, from_email=None, **kwargs)

        Sends an email to the user and returns the number of successfully
        delivered messages (which can be ``0`` or ``1`` since it can only send
        one message). If ``from_email`` is ``None``, Django uses the
        :setting:`DEFAULT_FROM_EMAIL`. Any ``**kwargs`` are passed to the
        underlying :meth:`~django.core.mail.send_mail()` call.

        .. versionchanged:: 3.1

            ``email_user()`` now returns the number of successfully delivered
            messages.
```

**Comment by felixxm** [comment]
Closing due to lack of follow up.

**Pull request closed without merging**